infrastructure/docker-compose.staging-deploy.yml

```diff
@@ -1,56 +1,86 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
-#
-# OpenCRVS is also distributed under the terms of the Civil Registration
-# & Healthcare Disclaimer located at http://opencrvs.org/license.
-#
-# Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
 version: '3.3'
 
 services:
+  notification:
+    environment:
+      - LANGUAGES=en,fr
+      - SENTRY_DSN=${SENTRY_DSN}
+      - QA_ENV=true
```

Comment:
@rikukissa if staging mimics production, then it should have 2FA and use the comms gateway.  Therefore QA should be false.  This environment should be exactly the same as production

Reply:
Shouldn't our current dev team change their conceptual understanding of using staging as a testing environment for new features?   Does this suggest that deploying to staging should be unrelated to merging a PR into develop as it currently is set up to do in workflows?  Instead it should be tied to releasing.

Reply:
Yea, you are correct, in our case we would deploy our release version to staging first and only then to demo. We do need to change the terminology a little and I think we should still have an environment to use as a latest development deployment / for tinkering with infrastructure setup. `dev` or `sandbox` or similar might work?

To answer your first comment, I purposely left out the renaming of environments out of this PR as we need to do the same for production as well.